### PR TITLE
README Refresh

### PR DESCRIPTION
**Pull request by andreamtp**
* Added list of supported SBCs
* Reported Fedora 36 status

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,23 @@
-# UEFI for PINE64 Quartz64 (Rockchip RK356X)
+# UEFI for Rockchip RK356X based SBR
```
Comment (jaredmcneill) [inline README.md]: What is `SBR`?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
 
-This repository contains a port of Tianocore EDK II to the PINE64 Quartz64 board.
+This repository contains a port of Tianocore EDK II to various Rockchip RK356x based boards:
 
-https://www.pine64.org/quartz64a/
+* [PINE64 Quartz64 A](https://www.pine64.org/quartz64a/) / [PINE64 Quartz64 B](https://www.pine64.org/quartz64b/) / [PINE64 SOQuartz](https://wiki.pine64.org/wiki/SOQuartz)
+* [Firefly roc-rk3566-pc](https://en.t-firefly.com/product/industry/rocrk3566pc.html) / [StationPC Station M2](https://www.stationpc.com/product/stationm2)
```
Comment (jaredmcneill) [inline README.md]: Firefly dev board names are stylized as `ROC-RK3566-PC`, etc.

File: README.md [hunk 3]
```diff
@@ -1,16 +1,23 @@
-# UEFI for PINE64 Quartz64 (Rockchip RK356X)
+# UEFI for Rockchip RK356X based SBR
 
-This repository contains a port of Tianocore EDK II to the PINE64 Quartz64 board.
+This repository contains a port of Tianocore EDK II to various Rockchip RK356x based boards:
 
-https://www.pine64.org/quartz64a/
+* [PINE64 Quartz64 A](https://www.pine64.org/quartz64a/) / [PINE64 Quartz64 B](https://www.pine64.org/quartz64b/) / [PINE64 SOQuartz](https://wiki.pine64.org/wiki/SOQuartz)
```
Comment (jaredmcneill) [inline README.md]: Lets match the vendor board naming, like `Quartz64 A` -> `Quartz64 Model A`, etc.

File: README.md [hunk 4]
```diff
@@ -1,16 +1,23 @@
...
+* [Firefly roc-rk3566-pc](https://en.t-firefly.com/product/industry/rocrk3566pc.html) / [StationPC Station M2](https://www.stationpc.com/product/stationm2)
+* [Firefly roc-rk3568-pc](https://en.t-firefly.com/product/industry/rocrk3568pc.html) / [StationPC Station P2](https://www.stationpc.com/product/stationp2)
```
Comment (jaredmcneill) [inline README.md]: For ROC-RK3568-PC, Station P2, and ROCK3 Compute Module, lets be clear that these are only available in source form at this time as the ports are works in progress.

File: README.md [hunk 5]
```diff
@@ -1,16 +1,23 @@
...
+If you want to build the image, checkout the repository and run:
+`$ make sdcard`
+
+Prebuild images are also provided and are available in the [release section](https://github.com/jaredmcneill/quartz64_uefi/releases).
 
 ## Running
 
-Connect a serial console to UART2 using settings 115200 8n1.
+After writing the image on your media of choice (eMMC, SD, M.2) according to the support available from your operating system of choice, is suggested to connect a serial console the UART2 of your board to see the boot.
+Please, rely on your vendor official documentation about the right settings, that usually are `115200 8n1` .
```
Comment (jaredmcneill) [inline README.md]: Lets leave the running section as-is. This UEFI port _always_ configures the serial console for 115200, which is almost always different from the default vendor settings.

**Review by jaredmcneill — CHANGES_REQUESTED** [review]
Thanks for the changes! Just a few suggestions..

File: README.md (hunk 1 above)
Comment (andreamtp) [reply README.md]: D'oh, typo: I mean SBC. Eventually can be used the extended format Single Board Computers

File: README.md (hunk 2 above)
Comment (andreamtp) [reply README.md]: ack

File: README.md (hunk 3 above)
Comment (andreamtp) [reply README.md]: ack

File: README.md (hunk 4 above)
Comment (andreamtp) [reply README.md]: Valuable input: haven't noticed that.

File: README.md (hunk 5 above)
Comment (andreamtp) [reply README.md]: ack

**Comment by andreamtp** [comment]
Added new commit with suggested improvements

**Review by jaredmcneill — APPROVED** [review]

**Comment by jaredmcneill** [comment]
LGTM, thanks!

**Pull request merged**